Crear pantalla “CategoriesScreen”.

SpectreLion opened the pull request:
La idea principal fue usar un compoente en el cual serviria para hacer
el listado de las categorias, sin embargo me dio problemas a la hora de
implementar lo del grid layout de reactive y lo coloque en una sola
pantalla, coloque en el package la dependecia necesaria para instalar, y
agregue al navigation al pantalla,junto con un color necesario.

SpectreLion committed:
Crear pantalla “CategoriesScreen”.

La idea principal fue usar un compoente en el cual serviria para hacer
el listado de las categorias, sin embargo me dio problemas a la hora de
implementar lo del grid layout de reactive y lo coloque en una sola
pantalla, coloque en el package la dependecia necesaria para instalar, y
agregue al navigation al pantalla,junto con un color necesario.

jaycorpstudios commented on package.json:
```diff
@@ -14,7 +14,8 @@
     "react-native": "0.52.2",
-    "react-navigation": "^1.0.0-beta.29"
+    "react-navigation": "^1.0.0-beta.29",
+    "react-native-layout-grid":  "0.0.2"
```
Para que necesitas esta dependencia? Puedes hacer el grid con simples estilos de Flex.

jaycorpstudios commented on src/screens/CategoriesScreen.js:
```diff
@@ -0,0 +1,108 @@
+      </TouchableHighlight>
+  );
+
+  /*goToTaskScreen(){
```
Si no vas a utilizar este código, es mejor borrarlo y no comentarlo.

jaycorpstudios commented on src/screens/CategoriesScreen.js:
```diff
@@ -0,0 +1,108 @@
+            style={styles.remindMeLogo}
+            source={require('./../images/remindMe-logo.png')}/>
+          <ScrollView style={styles.categoriesContainer}>
+            <GridLayout
```
Por favor haz el render del grid sin utilizar este componente, no es necesario para hacer un grid tan sencillo. Puedes utilizar una librería externa siempre y cuando le traiga un beneficio especial al proyecto, pero en este caso no le veo ninguno, o puedes explicarme por que sería tan necesario?

jaycorpstudios commented on src/screens/CategoriesScreen.js:
```diff
@@ -0,0 +1,108 @@
+  }
+}
+
+// TODO: Create Theme styles (Buttons, Global Components, etc) so it can be imported on each component as required.
```
Necesitas este comentario?

jaycorpstudios commented on src/screens/CategoriesScreen.js:
```diff
@@ -0,0 +1,108 @@
+  containerGrid: {
+    flex: 1,
+    flexDirection: 'column',
+    alligItems: 'center',
```
Tienes un error de sintáxis aquí, debería ser: "alignItems". No revisaste que funcionaran tus cambios antes de abrir tu Pull Request?

jaycorpstudios commented on src/App.js:
```diff
@@ -10,12 +10,13 @@ import {
     Home: { screen: HomeScreen },
-    Tasks: { screen: TasksScreen }
+    Tasks: { screen: TasksScreen },
+    Categories: {screen: CategoriesScreen}
```
A pesar de que la llamada al boton de Home esta actualizada y apunta a Categories, el orden aquí debería ser:

Home, Categories y finalmente Tasks.

jaycorpstudios commented on src/screens/HomeScreen.js:
```diff
@@ -33,7 +33,8 @@ export default class HomeScreen extends Component<{}> {
           style={styles.homeButton}
           underlayColor={Colors.touchableHover}
-          onPress={this.goToTaskScreen.bind(this)}>
+          //Cambiar dentro de la funcion goToCategoriesScreen el valor para probra la siguiente tarea
```
Es necesario este comentario aquí? Se que le estás diciendo al equipo como saltar a la otra pantalla, pero quien se va a encargar de eventualmente borrar este comentario si ya nadie va a tocar esta parte del código en ningún Pull Request futuro?

jaycorpstudios commented on src/screens/CategoriesScreen.js:
```diff
@@ -0,0 +1,108 @@
+  },
+  categoriesContainer: {
+    flex:1,
+    marginTop:20
```
Puedes probar con un margen para los lados laterales del contenedor? creo que se ve muy pegado el grid, o una vez que actualices tu grid, puedes asegurarte de dejar un espacio entre las categorías?

SpectreLion committed:
Correciones Pantalla "CategoriesScreen"

Se hicieron las correciones que mencionas, retomando la idea original de
crear un componente llamado "CategoriesList"

SpectreLion committed:
Cambio de nombre

Se cambio el nombre del componente y se quito el atributo resizeMode de la imagen de fondo

SpectreLion committed:
Merge branch 'develop' into WAKANDA-002
